app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::API
   include ActionController::HttpAuthentication::Token::ControllerMethods
 
   attr_reader :current_user
+  helper_method :current_user
```

Comment:
We still need the `attr_reader` as it is what creates the `current_user` method, and the `helper_method` makes the just-created method a helper method. My thoughts, though.